crates/aptos-indexer-api/src/main.rs

```diff
@@ -0,0 +1,41 @@
+mod graphql_root_queries;
+
+use aptos_indexer::database::new_db_pool;
+use async_graphql::{
+    http::{playground_source, GraphQLPlaygroundConfig},
+    EmptyMutation, EmptySubscription, Schema,
+};
+use async_graphql_poem::GraphQL;
+use clap::Parser;
+use graphql_root_queries::{ContextData, QueryRoot};
+use poem::{get, handler, listener::TcpListener, web::Html, IntoResponse, Route, Server};
+
+#[handler]
+async fn graphql_playground() -> impl IntoResponse {
+    Html(playground_source(GraphQLPlaygroundConfig::new("/")))
+}
+
+#[derive(Debug, Parser)]
+#[clap(author, version, about, long_about = None)]
+struct IndexerArgs {
+    /// Postgres database uri, ex: "postgresql://user:pass@localhost/postgres"
+    #[clap(long)]
+    pg_uri: String,
```

Comment:
afaik we pass this in via an env var in the indexer `INDEXER_DATABASE_URL`, would be good to continue in that way